R/stat_stepribbon.R

```diff
@@ -0,0 +1,77 @@
+#' Step ribbon statistic
+#'
+#' Provides stairstep values for ribbon plots
+#'
+#' @name stat_stepribbon
+#' @inheritParams ggplot2::geom_ribbon
+#' @param geom which geom to use; defaults to "`ribbon`"
+#' @param direction `hv` for horizontal-vertical steps, `vh` for
+#'   vertical-horizontal steps
+#' @references [https://groups.google.com/forum/?fromgroups=#!topic/ggplot2/9cFWHaH1CPs]()
+#' @return a ggplot
+#' @examples
+#' library(ggplot2)
+#'
+#' survival::survfit(survival::Surv(time, status) ~ 1, data = survival::lung) %>%
+#'   survival::survfit0() %>%
+#'   broom::tidy() %>%
+#'   ggplot(aes(x = time, y = estimate, ymin = conf.low, ymax = conf.high)) +
+#'   geom_step() +
+#'   geom_ribbon(stat = "stepribbon", alpha = 0.2)
+NULL
+
```

Comment:
can we have an example with visR::stat_stepribbon?

Reply:
added